packages/crawler/package.json

```
@@ -46,7 +46,7 @@
         "accessibility-insights-report": "2.0.0",
         "apify": "^0.21.3",
         "apify-shared": "^0.2.12",
-        "axe-core": "^3.5.3",
+        "axe-core": "4.0.1",
```

Should this have the caret? 
```suggestion
        "axe-core": "^4.0.1",
```